app/connectors/AuthConnector.scala

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2016 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package connectors
+
+import config.WSHttp
+import model.{BearerToken, LoginDetails, Role, SuccessfulAuthentication}
+import play.api.libs.json.Json
+import uk.gov.hmrc.play.config.ServicesConfig
+import uk.gov.hmrc.play.http._
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+
+object AuthConnector extends AuthConnector with ServicesConfig {
+  override val internalUserAuthBaseUrl: String = s"${baseUrl("auth")}/auth/authenticate/user"
```

Comment:
similarly this does not look to be internalUserAuthBaseUrl (not a base Url), may be rename it to authUrl?